internal/engine/upper.go

```diff
@@ -198,8 +198,11 @@ func handleBuildStarted(ctx context.Context, state *store.EngineState, action Bu
 		return
 	}
 
+	edits := []string{}
+	edits = append(edits, action.FilesChanged...)
+
 	bs := model.BuildRecord{
-		Edits:     append([]string{}, action.FilesChanged...),
+		Edits:     append(edits, ms.ConfigFilesThatCausedChange...),
```

Comment:
AFAICT, this introduces a bug where all subsequent builds will contain ContfigFilesThatCausedChange...

Reply:
Yep, you're right.